homeassistant/helpers/entity.py

```diff
@@ -253,7 +251,8 @@ def async_update_ha_state(self, force_refresh=False):
                             end - start)
 
         # Overwrite properties that have been set in the config file.
-        attr.update(get_overrides(self.hass, CORE_DOMAIN, self.entity_id))
+        if DATA_CUSTOMIZE in self.hass.data:
```

Comment:
Customize is only set when you use bootstrap to set it up. All the tests don't do it for example. 

If we make it mandatory it shouldn't live in data but on Hass. But it's too much a niche feature to be part of the core